src/table/constraint_system_metainfo.rs

```diff
@@ -91,15 +92,17 @@ impl<F: PrimeField> ConstraintSystemMetainfo<F> {
             0
         };
 
-        let poly = plonk::util::compress_expression(&exprs, challenge_index).expand();
+        let expr = plonk::util::compress_expression(&exprs, challenge_index);
+        let poly = expr.expand();
```

Comment:
NBD: if this is for backward compatibility, can add a TODO to remove it later.

Reply:
This is a draft, I will move this code to other branches sequentially and the last one will be with the deletion of all code concerning `Multipoly'